src/web/components/ClickToView.tsx

```diff
@@ -77,26 +78,51 @@ const roleButtonText = (role: RoleType) => {
 	}
 };
 
+const shouldDisplayOverlay = (
+	isTracking: boolean,
+	isOverlayClicked: boolean,
+	isInABTestVariant: boolean,
+) => {
+	if (!isTracking) {
+		return false;
+	}
+	if (isOverlayClicked) {
+		return false;
+	}
+	return isInABTestVariant;
+};
+
+const isInABTestVariant = (abTestConfig: CAPIType['config']['abTests']) => {
+	return abTestConfig.clickToViewVariant === 'variant';
+};
+
 export const ClickToView = ({
 	children,
 	role = 'inline',
 	onAccept,
 	isTracking,
 	source,
-	sourceDomain,
+	sourceDomain = 'unknown',
+	abTests,
 }: Props) => {
-	const [showOverlay, setShowOverlay] = useState<boolean>(true);
+	const [isOverlayClicked, setOverlayClicked] = useState<boolean>(false);
```

Comment:
There's a loose convention I think where for state hooks you have `thing`, `setThing` so maybe here we can do `isOverlayClicked`, `setIsOverlayClicked`? Or maybe use `overlayClicked`? But, to be honest, I can't think of a good reason why this should be so, other than this just seems to be where we've ended up with hook naming 🤷 , so this is very much a take it or leave it comment